src/Cimpress.Extensions.Http.Caching.InMemory/IMemoryCacheExtensions.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Threading.Tasks;
+using Cimpress.Extensions.Http.Caching.Abstractions;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace Cimpress.Extensions.Http.Caching.InMemory
+{
+    /// <summary>
+    /// Extension methods for an <see cref="IMemoryCache"/>.
+    /// </summary>
+    internal static class IMemoryCacheExtensions
```

Comment:
Except that the actual object we are extending in an IMemoryCache.  The namespace and the cache handler have different names then cache.

Thoughts?